src/content/blog/2024-04-10-the-telemetry-tale.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: ../../layouts/BlogLayout.astro
+title: The Telemetry Tale
+description: A journey into the metrics of Interledger
+date: 2024-04-10
+slug: the-telemetry-tale
+author: Sarah Jones
+author_url: https://www.linkedin.com/in/sarah-jones-ba6bb6b9
+tags:
+  - Interledger
+  - Telemetry
+---
+## A Journey into the Metrics of Interledger
+
+### Charting the Course
+
+Rafiki is open-source software that enables Account Servicing Entities (ASEs), like digital wallet providers, to leverage [Interledger](https://interledger.org/interledger) functionality. Interledger creates interoperability between different payment systems and currencies, making payments easier, faster, and cheaper. We hope that by removing the friction of this integration, we can expand the adoption of Interledger and grow the network. This is where our tale begins, on a balmy August in 2023, with the Rafiki team united for our work week in Cluj-Napoca, Romania. In order to measure the growth of the network, we needed to capture its state over time: 
+
+- how many transactions occurred on the network over the last week / month / year
+- how much value flowed over the network in the last week / month / year
+
+Since telemetry was marked as a high-priority item, promises were made about completing the feature in the course of the week. We envisioned a streamlined process using managed services. With a few configuration and integration tweaks, we’d be up and running in no time, right? Not quite, as we soon accepted—it is a truth universally acknowledged that software development timelines are more aspirational than absolute. Telemetry was merged into main 6 months later. This article provides an account of our journey and an explanation of our final solution.
```

Comment:
Very minor suggestion: Split into two sentences. "Not quite, as we soon accepted. It is a truth..."

Reply:
```suggestion
Since telemetry was marked as a high-priority item, promises were made about completing the feature in the course of the week. We envisioned a streamlined process using managed services. With a few configuration and integration tweaks, we’d be up and running in no time, right? Not quite, as we soon accepted. It is a truth universally acknowledged that software development timelines are more aspirational than absolute. Telemetry was merged into main 6 months later. This article provides an account of our journey and an explanation of our final solution.
```